docker/development/docker-compose.yml

```diff
@@ -81,20 +81,15 @@ services:
     environment:
       RAILS_ENV: development
       NODE_ENV: development
+      MAMPF_PORT: 3000
       # If you change these variables, make sure to adapt accordingly in
       # config/db-pgadmin.json
-      DEVELOPMENT_DATABASE_ADAPTER: postgresql
+      DATABASE_ADAPTER: postgresql
       DEVELOPMENT_DATABASE_DATABASE: mampf
       DEVELOPMENT_DATABASE_INTERACTIONS: mampf_interactions
       DEVELOPMENT_DATABASE_USERNAME: localroot
       DEVELOPMENT_DATABASE_HOST: db
       DEVELOPMENT_DATABASE_PORT: 5432
-      TEST_DATABASE_ADAPTER: postgresql
```

Comment:
It seems that the removal of these lines (and the changes in `database.yml`) results in a broken `rails db:up-reseed` from a fresh setup. I tried multiple times, the `rails db:drop` , `rails db:create` lines that are executed when restoring from the seed file fails because Rails does not get the database names right.

Reply:
Thanks, good catch, fixed in 62aaf9500fa